yeonjae/programmers-2/모음사전.js

```diff
@@ -0,0 +1,29 @@
+function solution(word) {
+  const dic = ["A", "E", "I", "O", "U"];
+  let checkValid = true;
```

Comment:
```suggestion
  let isValid = false;
```
- check보다는 boolean형인 isValid가 적절해보입니다.
- 갠적으로 (+) isValid라 함은 유효한지 여부를 판단하는 변수로 이해했는데 유효한데 answer를 증가시키는 것은 뭔가 의미상 싱크가 안맞는 것 같다고 느꼈습니다! 차라리 반대의 개념으로 접근하면 어떨까요

Reply:
is , check가 유효성체크할때 좀 고민을했는데 check라는 변수는 언제쓰는게 좀 좋을까요?! 
아무래도 불린형인경우는 is를 붙이는게 좀 적절한거겠죠?